### ISSUE TITLE
is it possible to switch the screen from Unity to Android(activity )or ios(viewcontroller)?

### ISSUE THREAD

**Issue by superbderrick** [Unity-Technologies/uaal-example#27]
Hi Unity forks

Due to changes in uaal function, Native->Unity->Native understands through samples

It’s very love it :)

But is it possible if we move from Unity screen to Android or each custom screen of ios?

Android
Unityplayeractivity -> nativeactivty
Back to unity

IOS
Unity viewcontroller-> native viewcontroller back to unity

Maybe it should be called as a plugin, right? , Or is it impossible?

Thanks
Best regards 
Derrick

**Comment by mnkyking** [comment]
@superbderrick Yes, the latest project does this review the iOS.md file and you'll see that it uses a iOS Native view then allows the user to initialize then show unity then through unity unload it to present the iOS Native view again.
